KubeArmor/core/karmorprobedata.go

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package core
+
+import (
+	"encoding/json"
+	cfg "github.com/kubearmor/KubeArmor/KubeArmor/config"
+	"log"
+	"os"
+)
+
+type KarmorData struct {
+	OSImage             string
+	KernelVersion       string
+	KubeletVersion      string
+	ContainerRuntime    string
+	ActiveLSM           string
+	KernelHeaderPresent bool
+	HostSecurity        bool
+	ContainerSecurity   bool
+	KubeArmorPosture    KubeArmorPostures
+}
+
+type KubeArmorPostures struct {
+	DefaultFilePosture             string
+	DefaultNetworkPosture          string
+	DefaultCapabilitiesPosture     string
+	HostDefaultFilePosture         string
+	HostDefaultNetworkPosture      string
+	HostDefaultCapabilitiesPosture string
+}
+
+func SetKarmorData(dm *KubeArmorDaemon) KarmorData {
+	var kd KarmorData
+	var postures KubeArmorPostures
+
+	if cfg.GlobalCfg.Policy && !cfg.GlobalCfg.HostPolicy {
+		kd.ContainerSecurity = true
+	} else if !cfg.GlobalCfg.Policy && cfg.GlobalCfg.HostPolicy {
+		kd.HostSecurity = true
+	} else if cfg.GlobalCfg.Policy && cfg.GlobalCfg.HostPolicy {
+		kd.ContainerSecurity = true
+		kd.HostSecurity = true
+	}
```

Comment:
```suggestion
	if cfg.GlobalCfg.Policy {
		kd.ContainerSecurity = true
	}
	if cfg.GlobalCfg.HostPolicy {
		kd.HostSecurity = true
	}
```

We can simplify the logic here!

Also let's move this logic inside if RuntimeEnforcer != nil. If there were trouble initalising the enforcer then no enforcement is involved.